src/applications/vaos/actions/newAppointment.js

```diff
@@ -150,6 +152,12 @@ export function startNewAppointmentFlow() {
   };
 }
 
+export function onClickedUpdateAddressButton() {
```

Comment:
Generally, we try to write these action creator function names as actions that are going to happen, so like `clickUpdateAddressButton`.

Reply:
Just need more practice and I'll get use the naming convention. Thanks!